core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -215,27 +230,37 @@ class LogCleaner(initialConfig: CleanerConfig,
   /**
    *  Abort the cleaning of a particular partition, if it's in progress. This call blocks until the cleaning of
    *  the partition is aborted.
+   *
+   *  @param topicPartition The topic and partition to be abort cleaning
    */
   def abortCleaning(topicPartition: TopicPartition): Unit = {
     cleanerManager.abortCleaning(topicPartition)
   }
 
   /**
    * Update checkpoint file to remove partitions if necessary.
+   *
+   * @param dataDir The file object to be updated
+   * @param partitionToRemove The topicPartition to be removed, default none
    */
   def updateCheckpoints(dataDir: File, partitionToRemove: Option[TopicPartition] = None): Unit = {
     cleanerManager.updateCheckpoints(dataDir, partitionToRemove = partitionToRemove)
   }
 
   /**
-   * alter the checkpoint directory for the topicPartition, to remove the data in sourceLogDir, and add the data in destLogDir
+   * Alter the checkpoint directory for the topicPartition, to remove the data in sourceLogDir, and add the data in destLogDir
```

Comment:
ScalaDoc allows to use backticks when you refer to variable names (or a paramers) in the doc.
```suggestion
   * Alter the checkpoint directory for the `topicPartition`, to remove the data in `sourceLogDir`, and add the data in `destLogDir`
```